apps/docs/pages/guides/storage/access-control.mdx

```diff
@@ -34,7 +34,27 @@ To allow read access to files, the RLS policy must allow users to `SELECT` the `
 
 Storage buckets are private by default. For private buckets, you can access objects via the [download](/docs/reference/javascript/storage-from-download) method. This corresponds to `/object/auth/` API endpoint. Alternatively, you can create a publicly shareable URL with an expiry date using the [createSignedUrl](/docs/reference/javascript/storage-from-createsignedurl) method which calls the `/object/sign/` API.
 
-For public buckets, you can access the assets directly without a token or an Authorisation header. The [getPublicUrl](/docs/reference/javascript/storage-from-getpublicurl) helper method returns the full public URL for an asset. This calls the `/object/public/` API endpoint internally. While no authorization is required for accessing objects in a public bucket, proper [access control](#access-control) is required for other operations like uploading, deleting objects from public buckets as well. Public buckets also tend to have [better performance](/docs/guides/storage-cdn#public-vs-private-buckets).
+For public buckets, you can access the assets directly without a token or an Authorisation header. The [getPublicUrl](/docs/reference/javascript/storage-from-getpublicurl) helper method returns the full public URL for an asset. This calls the `/object/public/` API endpoint internally. While no authorization is required for accessing objects in a public bucket, proper access control is required for other operations like uploading, deleting objects from public buckets as well. An easy way to get started would be to create RLS policies for `SELECT`, `INSERT`, `UPDATE`, `DELETE` operations and tightening the policies until it meets your security requirements. For example, one can start with the following `INSERT` policy:
+
+```sql
+create policy "policy_name"
```

Comment:
I don't think we should be recommending this even as a starting point. This will give users access to all buckets, even ones which are created in the future which is dangerous. 

Probably we can move this part to [this section](https://supabase.com/docs/guides/storage/access-control#policies) instead and link to that? Since this is relevant for both public and private buckets.

Reply:
updated the copy and moved it to the policies section